docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -141,6 +141,187 @@ which groups these values were observed, as well as the difference and ratio
 between the maximum and the minimum values. For more information refer to the
 subpackage :code:`fairlearn.metrics`.
 
+What traps can we fall into when modeling a social problem?
+--------------------------------------------------------------
+
+Machine learning systems used in the real world are inherently sociotechnical
+systems, which include both technologies and social actors. Designers of machine
+learning systems typically translate a real-world context into a machine learning
+model through abstraction: focusing only on 'relevant' aspects of that context,
+which are typically described by inputs, outputs, and the relationship between them.
+However, by abstracting away the social context they are at risk of falling into
+'abstraction traps': a failure to consider how social context and technology
+are interrelated.
+
+In this section, we explain what those traps are, and give some suggestions on
+how we can avoid them.
+
+In "Fairness and Abstraction in Sociotechnical Systems," Selbst et al. [#4]_
+identify failure modes that can arise from abstracting away the social context
+when modeling. They identify them as:
+
+* *The Solutionism Trap*
+
+* *The Ripple Effect Trap*
+
+* *The Formalism Trap*
+
+* *The Portability Trap*
+
+* *The Framing Trap*
+
+We provide some definitions and examples of these traps to help Fairlearn
+users think about how choices they make in their work can lead to or avoid
+these common pitfalls.
+
+The Solutionism Trap
+^^^^^^^^^^^^^^^^^^^^
+
+This trap occurs when we assume that the best solution to a problem
+may involve technology, and fail to recognize other possible solutions
+outside of this realm. Solutionist approaches may also not be appropriate
+in situations where definitions of fairness may change over time
+(see :ref:`formalism_trap`).
```

Comment:
```suggestion
(see :ref:`The Formalism Trap`).
```

Reply:
This should fix the reference!

Reply:
I'm confused. It's defined below (currently line 222), why wouldn't it work?

Reply:
Hmm, not sure. It's currently displaying as ":ref:`formalism_trap`" though

Reply:
Huh, that's weird... Did anybody figure out yet why this doesn't work?

Reply:
Applied suggestion. Let me know if the problem persists